pure_protobuf/dataclasses_.py

```diff
@@ -144,9 +159,30 @@ def message(cls: Type[T]) -> Type[TMessage]:
     return cast(Type[TMessage], cls)
 
 
+def make_one_of_field(field_: OneOf_, name: str) -> Dict[int, OneOfPartField]:
+    """
+    Figure out how to serialize and de-serialize oneof field.
+
+    Returns the corresponding ``Field`` instance.
+    """
+    child_fields = (
+        # TODO: what to do with packed?
```

Comment:
What's exactly the issue here? What'd happened if you just bypass the `packed` flag?

Reply:
> What's exactly the issue here?

There is no actually, just haven't known what to do and should it be exposed as part of api or not.

You mean pass it right into `part`?

Reply:
Yep. I guess the `packed` flag may be safely passed from the declaration through the part info to the `make_field` call